mappings/net/minecraft/util/UrlConstants.mapping

```diff
@@ -12,8 +12,15 @@ CLASS net/minecraft/unmapped/C_goxaugqo net/minecraft/util/UrlConstants
 	FIELD f_nsjugyzs BUY_JAVA_REALMS Ljava/lang/String;
 	FIELD f_pmlcwiug JAVA_LICENSES Ljava/lang/String;
 	FIELD f_qrdgtpsy JAVA_ATTRIBUTION Ljava/lang/String;
+	FIELD f_qwdxrntl MICROSOFT_SUPPORT Ljava/lang/String;
 	FIELD f_rijpagjo GDPR Ljava/lang/String;
+	FIELD f_sqgsrrfx PRIVACY_STATEMENT Ljava/lang/String;
 	FIELD f_tsimafix START_JAVA_REALMS_TRIAL Ljava/lang/String;
 	FIELD f_wgmbuhlh REALMS_CONTENT_CREATOR Ljava/lang/String;
 	FIELD f_wzgmmycp BUY_MINECRAFT_JAVA Ljava/lang/String;
 	FIELD f_xfpngjod SYMLINKS Ljava/lang/String;
+	METHOD m_bntwjjjx getExtendJavaRealmsUrl (Ljava/lang/String;Ljava/util/UUID;)Ljava/lang/String;
+		ARG 0 subscriptionId
+	METHOD m_raztlxpf getExtendJavaRealmsUrl (Ljava/lang/String;Ljava/util/UUID;Z)Ljava/lang/String;
+		ARG 0 subscriptionId
+		ARG 2 trialOrRealm
```

Comment:
I've not seen this style of naming booleans before, I assume:
- true => former
- false => latter
?
I'm not against it, but I'd like to make sure other find it intuitive.

Reply:
That is how it works, yeah. I took the method name from the Yarn mappings, but couldn't think of a better name for that parameter. Would just `trial` be better? It would then be:
`true` => trial
`false` => expired

There's really no truly intutive way to name it, at least that I can think of off the top of my head.

Reply:
nah, better as you have it